components/Services/BabyDegenPage/FurtherResources.jsx

```diff
@@ -1,56 +1,25 @@
-import { SUB_HEADER_CLASS, TEXT_CLASS } from 'common-util/classes';
-
-const { default: SectionWrapper } = require('components/Layout/SectionWrapper');
-const { MoveUpRight } = require('lucide-react');
+import { GetInvolvedCards } from 'components/GetInvolvedCards';
 
 const resources = [
   {
     title: 'Contribute a strategy',
     description:
       'Built a trading bot? Expert trader? Share your knowledge and add to the strategy library.',
-    action: {
-      url: 'https://discord.gg/BQzYqhjGjQ',
-      text: 'Reach out on Discord',
-    },
+    url: 'https://discord.gg/RHY6eJ35ar',
```

Comment:
let's use the previous link `https://discord.com/invite/BQzYqhjGjQ` it makes more sense to have an invitation from david